usrsctplib/netinet/sctp_pcb.h

```diff
@@ -317,6 +317,11 @@ struct sctp_base_info {
 	userland_thread_t recvthreadroute;
 #endif
 #endif
+
+#ifndef INVALID_SOCKET
+#define INVALID_SOCKET -1
+#endif
+
```

Comment:
Can we move that definition to `sctp_os_userspace.h`? We can also be specific and define it whenever we are no on Windows.